pycbc/__init__.py

```diff
@@ -45,7 +46,21 @@
 __version__ = pycbc_version
 
 
-def init_logging(verbose=False, format='%(asctime)s %(message)s'):
+_default_format = '%(asctime)s: %(message)s'
+
+
+class Formatter(logging.Formatter):
+    converter = dt.fromtimestamp
+
+    def formatTime(self, record, datefmt=None):
+        ct = self.converter(record.created).astimezone()
+        t = ct.strftime("%Y-%m-%dT%H:%M:%S")
+        s = "%s.%03d" % (t, record.msecs)
+        s += ct.strftime('%z')
+        return s
+
+
+def init_logging(verbose=False, fmt=_default_format):
```

Comment:
What motivates the change from `format` to `fmt`? This requires other changes in the code base (and is breaking the tests).

Reply:
I didn't want to overwrite the format builtin, but I now realise thats not needed